src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -0,0 +1,105 @@
+package com.snowflake.kafka.connector.streaming.iceberg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.snowflake.kafka.connector.Utils;
+import com.snowflake.kafka.connector.internal.DescribeTableRow;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class IcebergIngestionSchemaEvolutionIT extends IcebergIngestionIT {
+
+  @Override
+  protected Boolean isSchemaEvolutionEnabled() {
+    return true;
+  }
+
+  @Override
+  protected void createIcebergTable() {
+    createIcebergTable(tableName);
+  }
+
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("prepareData")
+  @Disabled
+  void shouldEvolveSchemaAndInsertRecords(
+      String description, String message, DescribeTableRow[] expectedSchema, boolean withSchema)
+      throws Exception {
+    // start off with just one column
+    List<DescribeTableRow> rows = describeTable(tableName);
+    assertThat(rows.size()).isEqualTo(1);
+    assertThat(rows.get(0).getColumn()).isEqualTo(Utils.TABLE_COLUMN_METADATA);
```

Comment:
We could have utilized the AssertJ fluent API here. Something like that should work

```suggestion
    assertThat(rows)
            .hasSize(1)
            .extracting(DescribeTableRow::getColumn)
            .containsExactly(Utils.TABLE_COLUMN_METADATA);
```

Reply:
nice idea